starknet_py/utils/crypto/transaction_hash.py

```diff
@@ -0,0 +1,96 @@
+from enum import Enum
```

Comment:
Do we want the methods in this module to be public? If so, let's add docstrings.